Update operations docs (#4369)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/operations/minio/index.md

To change the default MinIO cluster settings, create configuration file with the properties described above:
    
```yaml
cat <<EOF | kubectl apply -f -
apiVersion: v1
kind: ConfigMap
metadata:
  namespace: ${WORKSPACE_NAMESPACE}
  name: kaptain-overrides
data:
  values.yaml: |
    core:
      minio:
        servers: 4
        volumesPerServer: 4
        storageCapacity: 160Gi
EOF
```
This configuration will deploy 4 MinIO servers; each server will use 4 volumes with 10Gi capacity per volume.  
To learn more about MinIO distributed mode and replication see the [official documentation][minio-docs].

To install Kaptain with the provided parameters, refer to the [Deploy Kaptain documentation][deploy-kaptain].

## Expanding the MinIO cluster

The MinIO cluster can be expanded by increasing the number of server pools in the cluster. 

<p class="message--warning"><strong>WARNING: </strong>Expanding an existing server pool is not supported. The cluster can only by expanded by adding additional server pools.</p>

> To update an existing deployment of Kaptain, edit the `ConfigMap` with the custom configuration (if it was provided during the installation) or the default one 

Reviewer: ```suggestion
To update an existing deployment of Kaptain, edit the `ConfigMap` with the custom configuration you provided during the installation, or the default one:
```

```bash

> kubectl edit configmap -n <WORKSPACE_NAMESPACE> <name of the ConfigMap, e.g. kaptain-overrides or kaptain-2.0.0-defaults>

Reviewer: ```suggestion
kubectl edit configmap -n <WORKSPACE_NAMESPACE> <name of the ConfigMap, for example, kaptain-overrides or kaptain-2.0.0-defaults>
```

> kubectl edit configmap -n <WORKSPACE_NAMESPACE> <name of the ConfigMap, for example, kaptain-overrides or kaptain-2.0.0-defaults>

Reviewer 2: ```suggestion
kubectl edit configmap -n ${WORKSPACE_NAMESPACE} kaptain-overrides
```
thats what was used above.
